waltid-services/waltid-wallet-api/src/main/kotlin/id/walt/webwallet/web/controllers/AuthController.kt

```diff
@@ -123,17 +125,19 @@ fun Application.configureSecurity() {
                     defaultScopes = oidcConfig.oidcScopes
                 )
             }
-            urlProvider = { "${webConfig.publicBaseUrl}/wallet-api/auth/oidc-session" }
+            urlProvider = { "${oidcConfig.publicBaseUrl}/wallet-api/auth/oidc-session" }
         }
 
-        jwt("auth-oauth-jwt") {
-            realm = OidcLoginService.oidcRealm
-            // verifier(jwkProvider, oidcRealm)
-            verifier(OidcLoginService.jwkProvider)
+        if (FeatureManager.isFeatureEnabled(FeatureCatalog.oidcAuthenticationFeature)) {
+            jwt("auth-oauth-jwt") {
+                realm = OidcLoginService.oidcRealm
+                // verifier(jwkProvider, oidcRealm)
```

Comment:
since the file was reformatted, maybe the commented code can also be cleaned up?